README.md

```diff
@@ -118,12 +125,35 @@ On a new release, a new version is released to npm.
 
 ### Storybook
 
+Storybook is a frontend workshop for building UI components and pages in isolation https://storybook.js.org/. The Storybook can be used to develop and to test the components, but also to documentate the components and their features.
```

Comment:
documentate -> document (i.e. documentate is not a word, document is a valid verb)